python/lang/correctness/baseclass-attribute-override.py

```diff
@@ -12,8 +10,16 @@ def method2(self, args):
 
 class B:
     def method1(self, args):
-        print('hello there')
+        print("hello there")
```

Comment:
Should there be test cases for both single and double quoted strings?

Reply:
This case of single quotes doesn't really matter because the rule is about class inheritance, the print statement is not involved in the pattern.